keras/utils/np_utils.py

```diff
@@ -91,3 +91,47 @@ def normalize(x, axis=-1, order=2):
     l2 = np.atleast_1d(np.linalg.norm(x, order, axis))
     l2[l2 == 0] = 1
     return x / np.expand_dims(l2, axis)
+
+
+@keras_export("keras.utils.to_ordinal")
+def to_ordinal(y, num_classes=None, dtype="float32"):
```

Comment:
Small nitpick: can you move this up line 79 so that it's right after `to_categorical`?

Reply:
Moved this in the following commits